mbtools/copy_resources_s3.py

```diff
@@ -122,18 +134,28 @@ def main():
                         help='bucket name')
     parser.add_argument('s3_prefix', type=str,
                         help='prefix for s3 files')
+    parser.add_argument('-o', '--csv', const=None,
+                        help='hash to filename csv output file')
+
     args = parser.parse_args()
 
     new_resource_dir = Path(args.new_resource_path).resolve(strict=True)
 
     metadata_file = Path(args.metadata_file)
 
+    csv_file = None
```

Comment:
Minor: I don't think this line is necessary

Reply:
I currently use it so that either None or a Path Object is sent to upload_resources. I could also send the string args.csv to upload_resources and create the none or path object there... Are you thinking of a simpler way to get rid of this?

Reply:
D'oh! I see now, my mistake -- ignore this comment :joy: .